src/pages/About.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
-import {Box, Flex, Heading, Text, Link} from '@chakra-ui/react';
+import {Box, Flex, Heading, Text, Link, Image} from '@chakra-ui/react';
 import {useTranslation} from 'react-i18next';
 import {ExternalLinkIcon} from '@chakra-ui/icons';
+import cehatLogo from './CEHATLogo.png';
```

Comment:
I think the pages folder is the wrong place to put the image, maybe create a folder called "media" where we store all of the logos or something?